src/metabase/search/config.clj

```diff
@@ -186,57 +186,59 @@
   "All of the result components that by default are displayed by the frontend."
   #{:name :display_name :collection_name :description})
 
-(defmulti searchable-columns-for-model
-  "The columns that will be searched for the query."
-  {:arglists '([model])}
-  (fn [model] model))
+(defmulti searchable-columns
+  "The columns that can be searched for each model."
+  {:arglists '([model search-native-query])}
```

Comment:
Why no `?` in `search-native-query?`?

Reply:
Good q, I didn't like that it was inconsistent with `:search-native-query` in the search context

See
https://github.com/metabase/metabase/blob/f816a125fb265c4896e234427a7c3a5bf482145a/src/metabase/api/search.clj#L27-L39